src/main/java/com/ebay/myriad/scheduler/TaskFactory.java

```diff
@@ -120,7 +145,7 @@ private String getConfigurationUrl() {
                 //Place the hadoop config where in the HADOOP_CONF_DIR where it will be read by the NodeManager
                 //The url for the resource manager config is: http(s)://hostname:port/conf so fetcher.cpp downloads the
                 //config file to conf, It's an xml file with the parameters of yarn-site.xml, core-site.xml and hdfs.xml.
-                String configCopyCmd = "cp conf " + cfg.getYarnEnvironment().get("YARN_HOME") +
+                String configCopyCmd = "sudo cp conf " + cfg.getYarnEnvironment().get("YARN_HOME") +
```

Comment:
Unrelated change related to diagnosing an issue with remote distribution.  Will change back and do a separate PR.